server/autolinkplugin/command.go

```diff
@@ -12,13 +12,16 @@ import (
 	"github.com/mattermost/mattermost-plugin-autolink/server/autolink"
 )
 
+const template, pattern = "Template", "Pattern"
```

Comment:
2/5 if we parameterize command names we should do it for all commands, and use more intention-revealing names, perhaps `optTemplate` etc.

@iomodo we should file a ticket to convert autolink to autocomplete!

Reply:
It already has been filed https://github.com/mattermost/mattermost-plugin-autolink/issues/118